src/cascadia/WpfTerminalControl/TerminalTheme.cs

```diff
@@ -66,6 +66,16 @@ public struct TerminalTheme
         /// </summary>
         public uint DefaultForeground;
 
+        /// <summary>
+        /// The default selection background color of the terminal, represented in Win32 COLORREF format.
+        /// </summary>
+        public uint DefaultSelectionBackground;
+
+        /// <summary>
+        /// The opacity alpha for the selection color of the terminal, must be between 1.0 and 0.0.
+        /// </summary>
+        public float SelectionBackgroundAlpha;

```

Comment:
this isn't going to be an issue for VS, but... in the future, there will be more consumers for these things. How should we ensure that new fields don't break them?

Reply:
This is a good question to ask. I'm not sure what the best plan for this might be, and I don't want to maintain different versions of this API for every update we push. 

What is the design patter the rest of the Windows Console follows for API breaking changes?

Reply:
Our solution is _don't touch the Console API_. I believe we've started using a few unused bits in other flags in existing APIs, but we don't touch the console API.